app/components/metrics_summary_component/_index.scss

```diff
@@ -0,0 +1,36 @@
+.app-metrics__container {
+  margin-bottom: govuk-spacing(6);
+
+  @include govuk-media-query(tablet) {
+    display: grid;
+    grid-template-columns: repeat(3, minmax(govuk-px-to-rem(100), 1fr));
+    gap: $govuk-gutter;
+  }
+}
+
+.app-metrics__big-number {
+  @include govuk-typography-responsive(19);
+  @include govuk-typography-common;
+
+  padding: 5px 0;
+  margin-bottom: govuk-spacing(2);
```

Comment:
could we use govuk spacing? maybe using responsive spacing? lhttps://design-system.service.gov.uk/styles/spacing/#applying-spacing-in-your-own-css

Reply:
Yes, will use govuk-spacing here. Don't think it's worth using responsive spacing here at the moment, since the small spacing levels we're using here don't change between screen sizes in the responsive mixin.

Reply:
Done!